assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_uploadAll.vm

```diff
@@ -21,9 +21,17 @@
 			#set($downloadTemplateUrl="$accessPointUrl?contextString=" + $contextString + "&studentSubmissionText=true&studentSubmissionAttachment=true&gradeFile=true&feedbackTexts=true&feedbackComments=true&feedbackAttachments=true")
 			$tlang.getString('uploadall.instruction2')(<a href="$downloadTemplateUrl" title="$!tlang.getString('downall')">$tlang.getString('uploadall.instruction3')</a>) $tlang.getString('uploadall.instruction4')
 			<br />
-			<div class="highlight">
+			<p class="highlight">
 				$tlang.getString('uploadall.note')
-			</div>
+			</p>
+			#if ($plagiarismNote)
+				<p>
+					<img alt="$reviewIndicator" title="$reviewIndicator" src="/library/image/silk/rosette.png" class="assignmentRosette"/>
```

Comment:
Do we need some aria treatment on this `img`?

Reply:
To my understanding, alt is still preferred for images while aria-label is useful for other elements such as buttons.

The rosette image represents that the content will be submitted for content review.
The alt attribute is "Originality of submissions will be verified by {0}.", so the image's meaning is covered (though it might be redundant with the 'plagiarismNote' below)

Reply:
The plagiarismNote here is:
The submissions will be sent to {0} to be reviewed for plagiarism.
or
The submissions will be sent to {0} to be reviewed for plagiarism. They will be indexed to be compared against by other submissions.

I'd say these are redundant. I can add aria-hidden=true to hide the image from screenreaders.
Then for users without a visual disability, the icon serves as a quick visual indicator that content-review is enabled.
For other users, they'll see the icon and wonder "what's this?" - for these users the alt text is handy.

So I'm thinking, keep the alt text, and add aria-hidden=true. Does that sound reasonable?

Reply:
Sounds good to me, thanks Brian!